_posts/2022-09-26-opensearch-playground.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title:  "Launch Highlight: OpenSearch Playground"
+authors:
+  - liutaoaz
+  - andhopp
+date:   2022-09-26 00:00:00 -0700
+categories:
+  - community
+---
+
+We are excited to announce the live demo environment of OpenSearch and OpenSearch Dashboards. OpenSearch Playground provides a central location for existing and evaluating users to explore OpenSearch and OpenSearch Dashboards’ new and existing features without installation or download. Access it now here: [playground.opensearch.org](https://playground.opensearch.org).
```

Comment:
suggestion (2nd sentence):
"You can explore and try out new and existing features in OpenSearch and OpenSearch Dashboards without any installation. "

Reply:
Thanks, Mellissa have some comments for this too.